compiler/fir/fir2ir/src/org/jetbrains/kotlin/fir/backend/Fir2IrDeclarationStorage.kt

```diff
@@ -791,7 +791,6 @@ class Fir2IrDeclarationStorage(
                     isExternal = false,
                     isStatic = field.isStatic
                 ).apply {
-                    metadata = FirMetadataSource.Variable(field)
```

Comment:
Not 100% sure about this one, though it looks like it only affects imported Java fields and delegated "synthetic" fields, whatever that means, so it should not have any effect on emitted metadata.

That said, the fact that FIR fields, unlike descriptor fields, are not properties is slightly concerning, because e.g. they currently cannot be referenced. The IR simply doesn't allow it - it *requires* a property symbol for an `IrPropertyReferenceImpl`.